README.md

```diff
@@ -17,6 +17,13 @@ Requirements
 
 * [ZSH](http://zsh.sourceforge.net) 4.3 or newer
 
+Install
+------------------------------------------------------------------------------
+### Git
+* git clone https://github.com/zsh-users/zsh-history-substring-search.git
```

Comment:
I don't know how useful this is because it doesn't say where to clone the git repo nor what to do after cloning.  :scream_cat: So I would suggest that you omit this part and stick to the Homebrew instructions for now.